endaq/device/mqtt/manager.py

```diff
@@ -59,7 +59,7 @@
 # Maximum valid difference between device and system time. Times reported by
 # the device that differ from system time by this amount or more are
 # considered untrustworthy.
-MAX_DRIFT = 60 * 60 * 24 * 2
+MAX_DRIFT = 60 * 60
```

Comment:
Fine for now, but 1 hour still seems way too high, shouldn't it be more like 5 minutes? But I'm not 100% clear on how we get the NTP time